doc/developers/tips.rst

```diff
@@ -64,8 +64,24 @@ will be displayed as a color background behind the line number.
 Useful pytest aliases and flags
 -------------------------------
 
-We recommend using pytest to run unit tests. When a unit tests fail, the
-following tricks can make debugging easier:
+We recommend using pytest to run unit tests.
```

Comment:
I suspect with the dependence on parameterize, we should be more forceful than 'recommend'!

Reply:
I removed this sentence. After all, this is a section about pytest tips, so it already assumes that pytest is used.

The section about testing http://scikit-learn.org/dev/developers/contributing.html#testing-and-improving-test-coverage is explicit enough about the fact that pytest is needed.